pkgs/development/interpreters/janet/default.nix

```diff
@@ -1,28 +1,15 @@
 { lib, stdenv, fetchFromGitHub, meson, ninja }:
-
 stdenv.mkDerivation rec {
   pname = "janet";
-  version = "1.16.1";
+  version = "1.19.2";
 
   src = fetchFromGitHub {
     owner = "janet-lang";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-TzJbHmHIySlf3asQ02HOdehMR+s0KkPifBiaQ4FvFCg=";
+    sha256 = "sha256:0waj22rzxmc0yx1yr0pzw9lwp6my5abfpfi6vq932bmli8y9prpd";
   };
 
-  # we don't have /usr/bin/env in the sandbox, so substitute for a proper,
-  # absolute path to janet
-  postPatch = ''
-    substituteInPlace jpm \
-      --replace '/usr/bin/env janet' $out/bin/janet \
-      --replace /usr/local/lib/janet $out/lib \
-      --replace /usr/local           $out
-
-    substituteInPlace janet.1 \
-      --replace /usr/local/lib/janet $out/lib
```

Comment:
Is it safe to delete this?

Reply:
It was updating a man page, but the update doesn't really affect anything useful imo.

Reply:
What is the actual default value of `syspath`? `/usr/local/lib/janet` or `$out/lib`?

Reply:
It depends what you mean by default? If we change it is it really the default now?

Reply:
I mean `(dyn :syspath)` and it is `/nix/store/491jv1pqvlbkg04wjf933k6dka9snqrg-janet-1.19.2/lib/janet`, so the sentence should be like following:

```diff
- defaults to /usr/local/lib/janet on Linux/Posix, and C:/Janet/Library on Windows.
+ defaults to /nix/store/491jv1pqvlbkg04wjf933k6dka9snqrg-janet-1.19.2/lib/janet.
```

Reply:
I know, but if upstream wants that they should patch their own man page at install time, I don't know why you want to rewrite upstreams man pages. You could submit a patch to them if you want that.

Reply:
Ah, the sentence can be read as “the default value of the default value is ...”, then “/usr/local/lib/janet on Linux/Posix” makes sense even on NixOS.

Reply:
That was my interpretation, I guess I was not so clear rereading.